sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/implementation/TransactionalBatchImpl.java

```diff
@@ -52,7 +52,7 @@ public TransactionalBatchImpl(AzureTableImpl client, SerializerAdapter transacti
      */
     @Host("{url}")
     @ServiceInterface(name = "AzureTableServices")
-    private interface TransactionalBatchService {
+    public interface TransactionalBatchService {
```

Comment:
Was this done by hand or was there an update to the Swagger configurations? If the former this will be overwritten every time the client is regenerated.

Reply:
This was done by hand because this particular interface was also written by hand. The others were already being generated with `public` due to the swagger markdown file being configured as such.